pufferpanel

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env bash
 
-red=$(tput setf 4)
-green=$(tput setf 2)
-yellow=$(tput setf 6)
-blue=$(tput setf 1)
+red=$(tput setaf 1)
```

Comment:
oh ok

On Sat, Sep 10, 2016 at 7:21 PM, Joshua Taylor <notifications@github.com>
wrote:

> In pufferpanel
> <https://github.com/PufferPanel/PufferPanel/pull/712#discussion_r78285491>
> :
>
> > @@ -1,9 +1,9 @@
> >  #!/usr/bin/env bash
> >
> > -red=$(tput setf 4)
> > -green=$(tput setf 2)
> > -yellow=$(tput setf 6)
> > -blue=$(tput setf 1)
> > +red=$(tput setaf 1)
>
> As this is already included on master, please don't add this change as
> well. It will make the merge worse for us.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/PufferPanel/PufferPanel/pull/712/files/5dc2aa51892a2f8a3077e3cadf16a5d5617e0430#r78285491>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AICNx34o88wf_KXxCSnIsJthSarcjoezks5qozuHgaJpZM4J525Q>
> .
>